js-miniapp-bridge/src/common-bridge.ts

```diff
@@ -631,6 +632,22 @@ export class MiniAppBridge {
       );
     });
   }
+
+  /**
+   * @param alertInfo Close confirmation alert info.
+   * @returns Promise resolves with the contact id received a message.
```

Comment:
I think, we can remove `contact id..` from this comment.

Reply:
🤦🏻  My bad. Nice catch.